components/level/Player.module.css

```diff
@@ -1,6 +1,160 @@
 .classic {
   border-color: var(--level-player-extra);
   border-width: 1px;
+  color: transparent !important;
+}
+.classic::before, .classic::after {
+  --block-size: 2em;
+  --line-size: calc(var(--block-size) * 0.05);
+  animation: 1s linear infinite rotation;
+  animation-play-state: inherit;
+  animation-delay: 1s;
+  content: "";
+  position: absolute;
+}
+.classic::before {
+  /* before = lower layer = line spinner */
+  --line-length: calc(var(--block-size) * 1.4142); /* make line match diagonal length *sqrt(2) */
+  height: var(--line-size);
+  width: var(--line-length);
+  top: calc((var(--block-size) - var(--line-size)) / 2); /* vertically centered */
+  left: calc((var(--block-size) - var(--line-length))/2); /* horizontally centered */
+  background-color: var(--level-grid-used);
+}
+.classic::after {
+  /* after= upper layer = "circle" spinner, split into 24 segments for ease of path scaling */
+  --circle-size-factor: 0.8;
+  --diameter: calc(var(--block-size) * var(--circle-size-factor));
+  --circle-offset: calc((var(--block-size) - var(--diameter))/2);
+  --inner-circle-radius: 15%;
+  height: var(--diameter);
+  width: var(--diameter);
+  top: var(--circle-offset);
+  left: var(--circle-offset);
+  border-radius: 50%; /* allows drawing lines to bounding box and clips them away to within the circle */
+  /* gradient to make an inner "knob" with spokes and outer ring effect */
+  background: radial-gradient(circle, #eee 5%, #ccc calc(var(--inner-circle-radius)*0.8), #555 calc(var(--inner-circle-radius)/5*7), #999 61%, #eee 66%, #999 71%);
+  
+  /* goneometry helper values */
+  --sin0: calc(0);
+  --sin15: calc(0.2588);
+  --sin30: calc(0.5);
+  --sin45: calc(1.4142 / 2);
+  --sin60: calc(1.7321 / 2);
+  --sin75: calc(0.9659);
+  --sin90: calc(1);
+  --cos0: var(--sin90);
+  --cos15: var(--sin75);
+  --cos30: var(--sin60);
+  --cos45: var(--sin45);
+  --cos60: var(--sin30);
+  --cos75: var(--sin15);
+  --cos90: var(--sin0);
+  /* we're resorting to a polygon approximation over the generic SVG-path() option as the latter seems not easily scalable */
```

Comment:
Quick q: why would the svg not be scalable?

Reply:
As far as I could find, when specifying a plain SVG path() the coordinates in it are using absolute values (pixels) compared to the element's bounding box (which is used as the viewbox for the SVG path).
From what I find, it might work by creating the SVG as an embedded SVG in the html and then using the `url(#id)` syntax for it. I'll see if I can get around to that approach sometime. According to [this example](https://lab.iamvdo.me/css-svg-masks/#testC5) you can then have the clipping bounds set as the relative object bounds, which would be perfect.